{
	"name": "CVE-2019-14433",
	"description": "An issue was discovered in OpenStack Nova before 17.0.12, 18.x before 18.2.2, and 19.x before 19.0.2. If an API request from an authenticated user ends in a fault condition due to an external exception, details of the underlying environment may be leaked in the response, and could include sensitive configuration or other data.",
	"packages": [
		{
			"name": "nova",
			"releases": {
				"bullseye": {
					"status": "resolved",
					"repositories": {
						"bullseye": "2:22.0.1-2"
					},
					"fixed_version": "2:19.0.2-1",
					"urgency": "low"
				},
				"buster": {
					"status": "open",
					"repositories": {
						"buster": "2:18.1.0-6"
					},
					"urgency": "low",
					"nodsa": "Minor issue",
					"nodsa_reason": "",
					"next_point_update": true
				},
				"sid": {
					"status": "resolved",
					"repositories": {
						"sid": "2:25.0.0-1"
					},
					"fixed_version": "2:19.0.2-1",
					"urgency": "low"
				},
				"stretch": {
					"status": "open",
					"repositories": {
						"stretch": "2:14.0.0-4+deb9u1",
						"stretch-security": "2:14.0.0-4+deb9u1"
					},
					"urgency": "low",
					"nodsa": "Minor issue",
					"nodsa_reason": ""
				}
			}
		}
	],
	"scope": "local"
}